cluster/operations/untrusted-worker.yml

```diff
@@ -28,6 +31,7 @@
       name: garden
       properties:
         garden:
+          forward_address: 127.0.0.1:7777
```

Comment:
I don't think this does anything. We should probably remove it from the other manifest.